src/layout/FeaturesSection/FeaturesSection.scss

```
@@ -8,7 +8,7 @@
 		// Inner section
 		.page-section-inner {
 			display: grid;
-			grid-template-columns: calc(50% - 24px) calc(50% - 24px);
+			grid-template-columns: calc(25%% - 24px) calc(25% - 24px) calc(25% - 24px) calc(25% - 24px);
```

this can be shortened down to
```suggestion
			grid-template-columns: repeat(auto-fill, calc(1fr - 24px));
```